qiskit/transpiler/passes/synthesis/high_level_synthesis.py

```diff
@@ -187,9 +211,13 @@ def run(self, dag: DAGCircuit) -> DAGCircuit:
                 else:
                     plugin_method = plugin_specifier
 
-                # ToDo: similarly to UnitarySynthesis, we should pass additional parameters
-                #       e.g. coupling_map to the synthesis algorithm.
-                decomposition = plugin_method.run(node.op, **plugin_args)
+                decomposition = plugin_method.run(
+                    node.op,
+                    coupling_map=self._coupling_map,
+                    target=self._target,
+                    qubits=[dag_bit_indices[x] for x in node.qargs],
```

Comment:
There's actually a new interface for this added in 0.25, you can do:

```suggestion
                    qubits=[dag.find(x).index for x in node.qargs],
```

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/10477/commits/11f92363af5b536a0b0ce3444e4e2795b70419e9. But the method is called `find_bit`, not `find` :p (this is supposed to be sticking-tongue-out-emoji).

Reply:
Oops my bad, that's what I get for reviewing at the end of the day. I didn't notice my own typos :)